rootfs/usr/lib/frzr.d/install-0001-require_reboot.migration

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+# triggers notification for reboot in old BPM
```

Comment:
Is this still required? Old BPM was removed from the client like a year ago

Reply:
> Is this still required? Old BPM was removed from the client like a year ago

I have no idea. I found it causing no harm and so I left it sitting there.